pytoml/parser.py

```diff
@@ -203,11 +204,16 @@ def _p_basicstr_content(s, content=_basicstr_re):
     return ''.join(res)
 
 _key_re = re.compile(r'[0-9a-zA-Z-_]+')
+_quotes_re = re.compile(r'[\'"]')
 def _p_key(s):
     with s:
-        s.expect('"')
-        r = _p_basicstr_content(s, _basicstr_re)
-        s.expect('"')
+        q = s.expect_re(_quotes_re)
+        if q.group(0) == "'":
+            r = _p_basicstr_content(s, _literalstr_re)
```

Comment:
This will process escapes, which is not what you want. You should simply consume `_litstr_re`.